### ISSUE TITLE
Can't get it to work

### ISSUE THREAD

**Issue by pixelass** [SamVerschueren/vscode-linter-xo#32]
VS CODE: Version 1.13.1 (1.13.1)
OS X 10.12.5 (16F73)

I activated the plugin as suggested.

```json
{
	"stylelint.enable": true,
    "xo.enable": true,
    "css.validate": false,
    "scss.validate": false,
    "less.validate": false
}
```

```json
{
  "name": "foobar",
  "devDependencies": {
      "xo": "^0.18.2"
   },
  "xo": {
    "tabs": false
  }
}
```

I see nothing happening at all.

**Comment by pixelass** [comment]
Seems to be related to https://github.com/avajs/ava/issues/1418 which mentions an issue with Ava and XO in large projects.

I was able to get it to work by adding a second package.json in a deeper folder (we only lint this one).

This is very ugly and prevents a regular workflow.
It seems that there is a major issue in XO and Ava or one of their shared dependencies. 

I tested it in a smaller scope and in my test-repo for XO and Ava (linked in github issue above)

Both worked. 


CLOSING
